ci-operator/step-registry/cucushift/installer/check/control-plane-machinesets/cucushift-installer-check-control-plane-machinesets-commands.sh

```diff
@@ -79,3 +82,35 @@ fi
 
 echo "control-plane machinesets:"
 cat "${stdout}"
+
+# control-plane machine name check
+# Machines
+err_output=$(mktemp)
+machine_output=$(mktemp)
+oc get machine -n openshift-machine-api --selector machine.openshift.io/cluster-api-machine-type=master --no-headers -owide 1>${machine_output} 2>${err_output}
+
+if grep -ir 'No resources found in openshift-machine-api namespace.' ${err_output}; then
+    echo "ERROR: No machines in openshift-machine-api namespace, but found controlplanemachinesets!"
+    exit 1
+fi
+
+control_plane_nodes_count=$(cat "${machine_output}" | wc -l || true)
+excepted_count=$(cat "${machine_output}" | awk '{print $1}' | grep -iE "master-[0-9]{1}$" | wc -l || true)
+
+echo "control_plane_nodes_count: ${control_plane_nodes_count}"
+echo "excepted_count: ${excepted_count}"
+
+if (( ${excepted_count} < 1 )) || (( ${control_plane_nodes_count} < 1 )); then
+    echo "ERROR: control plane nodes count or expected nodes count is less than 1, exit now."
+    exit 1
+fi
+
+if [[ "${excepted_count}" != "${control_plane_nodes_count}" ]]; then
+    echo "ERROR: One or more control plane machine name is not expected."
+    exit 1
```

Comment:
From the [log](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/43094/rehearse-43094-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-azure/1699683248948383744):
```
control_plane_nodes_count: 4
excepted_count: 1
ERROR: One or more control plane machine name is not expected.
```

I think we should add the following lines to some prior place for easy debugging.
```
echo "Machines:"
cat "${machine_output}"
```